src/cms/templates/users/region/user.html

```diff
@@ -88,13 +88,22 @@ <h3 class="heading font-bold text-white">{% trans 'Extended Settings' %}</h3>
         </div>
 	    {% if user_form.instance.id %}
 	    <div class="pt-2 pb-4">
-		    <button class="bg-red-500 hover:bg-red-600 text-white font-bold py-3 px-4 rounded" onclick="confirmation_popup(event, '#confirm_delete_user')">
-			    {% trans 'Delete this user' %}
-		    </button>
-
-		    {% include "../confirmation_popups/delete_region_user.html" %}
+            <button title="{% trans 'Delete user' %}" class="confirmation-button w-full bg-red-500 hover:bg-red-600 text-white font-bold py-2 px-4 rounded"
+                    data-confirmation-title="{% trans 'Please confirm that you really want to delete this user' %}"
+                    data-confirmation-text="{% trans 'This cannot be reversed.' %}"
+                    data-confirmation-subject="{{ user_form.username.value }}"
+                    data-action="{% url 'delete_region_user' region_slug=region.slug user_id=user_form.instance.id %}">
+                <i data-feather="trash-2" class="inline-block mr-2"></i>
+                {% trans 'Delete this user' %}
+            </button>
```

Comment:
If the user also has other regions, the button should state something like "Remove user from region" instead of "Delelte user" and the text in the confirmation popup should be adapted.

Reply:
You're right, I added two versions of the button.